README.md

```diff
@@ -74,11 +75,15 @@ spec:
   instance:
     name: "production"              # name of the InfobloxInstance
   networkView: "datacenter-network" # Infoblox network view that will be used
+  dnsView: "some-dns-view"          # <--- NEW: DNS view for this pool (optional)
   subnets:                          # list of the subnets in the network view we want to get IP addresses from
     - cidr: "10.0.0.0/24"           # subnet CIDR
       gateway: "10.0.0.1"           # gateway that should ba assigned to the IP Address claim
 ```
 
+- If `dnsView` is not set, the pool will inherit the `defaultDnsView` from the referenced InfobloxInstance.
+- If neither is set, the provider will use the Infoblox default DNS view.
```

Comment:
There are now two sections in the readme that talk about DNS. Down below you've also added basically the same information. Can we just remove this part here?